Modules/Devices/src/Devices.Application/PushNotifications/Commands/UpdateDeviceRegistration/UpdateDeviceRegistrationCommand.cs

```diff
@@ -7,5 +7,5 @@ public class UpdateDeviceRegistrationCommand : IRequest<UpdateDeviceRegistration
     public string Platform { get; set; }
     public string Handle { get; set; }
     public string AppId { get; set; }
-    public string? Environment { get; set; }
+    public string Environment { get; set; }
```

Comment:
Can you keep the ? here please? Currently we don't have null checks enabled yet. But as soon as we have, this will be a thing.
And we will enable it once you're done with your current tasks.